lib/round.rb

```diff
@@ -0,0 +1,145 @@
+class Round
+
+  attr_accessor :deck, :turns
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+  end
+
+  def current_card
+    deck.cards.first
+  end
+
+  def take_turn(guess)
+    turn = Turn.new(guess, deck.cards.first)
+    turns << turn
+    deck.cards.rotate!
+    return turn
+  end
+
+  def correct?
+    round.turns.first.guess == round.turns.fist.card.answer
+  end
+
+  def number_correct
+    num_correct = 0
+    turn_index = 0
+    until turn_index == @turns.count
+      turns.each do |turn|
+        if turns[turn_index].guess == turns[turn_index].card.answer
```

Comment:
I see you're directly comparing the guess to the answer - this will determine if the guess _exactly matches_ the answer; an edge case here that should be accounted for is if the case is off. For instance if I type "Justin Bieber", it is no less correct than "justin bieber" which is what you have in your `cards.txt` file.